interface/wx/image.h

```diff
@@ -321,6 +321,14 @@ class wxImageHandler : public wxObject
     */
     void SetName(const wxString& name);
 
+    /**
+       Sets the bitmap type for the handler.
+
+       @param mimetype
+           The bitmap type.
+    */
+    void SetType(wxBitmapType type);
```

Comment:
I'm not sure if we want to make this one public neither... IMO it should be protected in the best case, it doesn't make any sense to change `wxImageHandler` type from the outside, does it?

Reply:
Probably not, but all the other setters are public in the interface as well as the real class, which is likely why I put it there. I think this one was requested by a user who was creating their own handler from Python so it probably needs to stay, but making it (and the other setters?) protected is fine with me.

Reply:
Added http://trac.wxwidgets.org/ticket/18082 to determine what to do about SetType and other setters.